### ISSUE TITLE
Term Request: Corneal ectasia

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11770]
**Preferred term label:**
Corneal ectasia

**Synonyms:**
* Keratectasia
* Corneal ectatic disorder
* Ectasia of the cornea
* Secondary corneal ectasia

**Definition (free text):**
A progressive non-inflammatory structural disorder of the cornea characterized by localized or generalized stromal thinning and steepening (protrusion) of the corneal tissue. This biomechanical failure leads to irregular astigmatism, increased corneal curvature, and decreased visual acuity, occurring either as a primary disorder (such as keratoconus or pellucid marginal degeneration) or secondary to corneal refractive surgery or trauma.

**Parent term (use hpo.jax.org/app):**
Decreased corneal thickness [HP:0100689])

**Your nano-attribution (ORCID):**
0000-0001-9042-0508
